config/default.json

```diff
@@ -30,8 +30,10 @@
         "enabled": true, // you must send the bot a message once.
         "queuePlace": 20
     },
+    "antiAntiAFK": { 
+        "enabled": true, // master switch for all bypass antiAFK plugins
```

Comment:
the trailing comma causes a crash

Reply:
Okey, it wasn't failing with npm start, so tests are parsing json differently?  Anyway, I accidentally fixed it in commit with antiafk configuration :)